campaigns.json

```diff
@@ -2705,16 +2705,25 @@
     "go": true,
     "lumisize": -1,
     "maxcopies": 1,
-    "SecondaryLocation": [
-       "T1_IT_CNAF_Disk",
-       "T1_US_FNAL_Disk"
-    ],
     "SiteBlacklist": [
        "T2_CH_CERN",
        "T2_CH_CERN_HLT"
     ],
     "secondaries": {
-      "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL18_106X_upgrade2018_realistic_v16-v1/PREMIX": {}
+      "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL18_106X_upgrade2018_realistic_v16-v1/PREMIX": 
```

Comment:
This one already has copies at CNAF and FNAL and these are powerful sites. So, this selection look good to me.

Reply:
Remember that the first Neutrino sample already went through this we are only really adding the second one.

Reply:
Okay, perfect